README.md

```diff
@@ -12,24 +12,51 @@ anyone can use this project to run said accelerators on the cloud and
 develop host code (such as drivers and userspace libraries) targetting
 said accelerators.
 
+# Overview
+
+This project consists of three main parts.
+
+## QEMU Model
+
+This part of the project will develop and ideally upstream a
+hermes-based PCIe device model. This code will, initially, be hosted
+in a public fork of [QEMU][4]. Some early work on such a model results
```

Comment:
Our QEMU fork is currently private. If we make it public, then we should update this link to point to it.

Reply:
@iomartin agreed but for now let's keep it private.